### ISSUE TITLE
OpenVehicleVision  database

### ISSUE THREAD

**Issue by yeyang1021** [baidut/OpenVehicleVision#62]
I want to know , how can I  to download the database for the algorithm. I couldn't download them.

**Comment by baidut** [comment]
I have uploaded all related database to my cloud disk. To download from my cloud disk:
1. open http://yun.baidu.com/share/link?shareid=1552701070&uk=1863129434
2. click `项目公开`, then -> `OpenVehicleVision` and  -> `datasets` 

You can also download the database from website

# SLD2011

**Santaigo Lanes Dataset (SLD2011)** [website](http://ral.ing.puc.cl/datasets/ldw/)

Related paper：
[A comparison of gradient versus color and texture analysis for lane detection and tracking](http://ieeexplore.ieee.org/xpls/abs_all.jsp?arnumber=5418326)

Related video：
[Santaigo Lanes Dataset (SLD2011) - Lane changing maneuvers at dusk]   [youtube](https://www.youtube.com/watch?v=4MSNXvYYR2Y)
Lane tracking with shadows 2 - Bird Eye View - Santaigo Lanes Dataset (SLD2011) [youtube](https://www.youtube.com/watch?v=kHDMUQWIadk)

# ROMA (ROad MArkings)

website: http://www.lcpc.fr/english/products/image-databases/article/roma-road-markings-1817
related paper: Evaluation of Road Marking Feature Extraction [view on IEEE.org](http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=4732564) 

# KITTI-ROAD dataset

[Road/Lane Detection Evaluation 2013](http://www.cvlibs.net/datasets/kitti/eval_road.php)

Please note the license conditions of this software / dataset!

[youtube video](https://www.youtube.com/watch?v=KXpZ6B1YB_k)
- `kitti/data_road.zip` [base kit with: left color images, calibration and training labels (0.5 GB)](http://kitti.is.tue.mpg.de/kitti/data_road.zip)
- `kitti/data_road_right.zip` [right color image extension (0.5 GB)](http://kitti.is.tue.mpg.de/kitti/data_road_right.zip)
- `NA` grayscale image extension (0.3 GB)
- `NA` Velodyne laser point extension (1 GB)
- `NA` OXTS GPS/IMU extension (1 MB)
- `kitti/devkit_road.zip` [development kit (1 MB)](http://kitti.is.tue.mpg.de/kitti/devkit_road.zip)
- `kitti/devkit_road_mapping.zip` [Mapping of training set to raw data sequences (1 MB)](http://kitti.is.tue.mpg.de/kitti/devkit_road_mapping.zip)

**Comment by yeyang1021** [comment]
thx.

**Comment by yeyang1021** [comment]
ROMA (ROad MArkings)

website: http://www.lcpc.fr/english/products/image-databases/article/roma-road-markings-1817
related paper: Evaluation of Road Marking Feature Extraction view on IEEE.org

I can not open the url, and I can not find the original image about the ROMA datasets in baiduyun.
Only find the groundtruth images.
Could you upload the datasets in baiduyun.

**Comment by baidut** [comment]
Thank you for your feedback.

I have uploaded the raw roma dataset `OpenVehicleVision/datasets/ROMA/roma.zip`
which includes original images (.jpg) and road marking groundtruth (.pgm).

We manually labeled road area ground truth of that dataset, see  `OpenVehicleVision/datasets/ROMA/ROMA_ROAD_GT.zip`

Note ROMA dataset is for research use only and you should cite their paper if you use them.
Also, I'd appreciate it if you cite my paper.

```
@INPROCEEDINGS{Zhenqiang2016,
author={Zhenqiang Ying and Ge Li}, 
booktitle={Acoustics, Speech and Signal Processing (ICASSP), 2016 IEEE International Conference on}, 
title={Robust Lane Marking Detection using Boundary-Based Inverse Perspective Mapping}, 
year={2016}, 
intype = {to appear in},}
```

Our paper has not published yet, I'd like to give you a copy if you need.

**Comment by yeyang1021** [comment]
Thank you for your dataset.
There are still some problems in the code(Matlab).
When run the ICASSP2016.m, the roadDetectionViaBird.m has a error due to the imdump function is missing.  You can check the error in the 77 row. Maybe you not provide it.
I want to see your paper, if you are convenient. Email: yeyang1021@163.com

**Comment by Spir4L** [comment]
Hi,  your package is really nice and wel done! By the way I am trying to use your roadDetectionViaBird fucntion on the Kitti dataset but I find that it works badly than on the other ones. Do you have any idea of why?

**Comment by baidut** [comment]
@Spir4L Thank you for your reply! 
Kitti dataset  is very challenging, I suggest you try FSO:

Project http://abhijitkundu.info/projects/fso/
Code https://bitbucket.org/infinitei/videoparsing

Or use deep learning

Multi-scale context aggregation by dilated convolutions. https://github.com/fyu/dilation

I am still in the holidays, and I will update the code during the school.
Sorry to delay your time! Hope it'll help you!

**Comment by Spir4L** [comment]
@baidut  Thank you very much. I will have a look at them. 
Have a nice day!

**Comment by GloriaHM** [comment]
hello, I can not download the Velodyne laser point  database from the websites. I have tried many times. You  cloud disk link is Invalid. Can you update the link again? Thank you for your help.
